### ISSUE TITLE
English Textpack marked for update when other languages amended

### ISSUE THREAD

**Issue by philwareham** [textpattern/textpattern#1170]
While I've been adding additional translations for other language Textpacks (not English), I've noticed that when I log in to the control panel of my test site the English (my base language install) Textpack is highlighted as 'Update' when it shouldn't - because only other languages have been updated. Any idea why this is happening @Bloke ?

**Comment by Bloke** [comment]
Interesting. Lemme look at that later tonight.

**Comment by philwareham** [comment]
I mean, it's no big problem - if I press update, or don't press it, the Textpack is fine either way. Once update is pressed the alert goes away too. Not sure if this happens on other languages if they are set to base, but I assume it does.

**Comment by phiw13** [comment]
>  Not sure if this happens on other languages if they are set to base, but I assume it does.

Nope, or at least, not on my side (`en-gb` as base, `fr` on the admin side). 2 of the installed languages  were updated this AM.

**Comment by Bloke** [comment]
I can't reproduce this. I updated English from the UI, altered a few lang files from other languages I already had installed and uploaded the files. They showed as 'update', English did not.

Are there specific steps to reproduce this? Or pre-requisites like a language being (or not being) installed already? Or the public/admin UI set to a particular language when the files are replaced? Or running local vs running online?

I vaguely think I might have seen something similar over the past few months and just put it down to "Hmmm, must have been an update to that file as well then") but I never really looked into it. The only time the language should show as updated is if the timestamp of the uploaded file for that language differs from the corresponding last installed timestamp in the database (regardless if the contents was changed).
